### DISCUSSION TITLE
Mesh size for thin objects

### DISCUSSION THREAD

**Discussion by eryd** [firemodels/fds#12684] [Q&A]
All, I have a question about the treatment of thin objects in FDS.

I am developing a model with shading device made of wood planks. Each plank acts as a vertical shading to the building. The planks are arranged side by side, with 0.5m spacing between them. The FDS simulation will study the fire propagation from the exterior to the inside.

The size of plank is 0.3m depth, and up to 10m long (vertical). The thickness is 3cm.

The obstruction and the surface are something like the following:

```
&OBST ID='New OBST' SURF_ID='Wood'
	  XB=0.000,0.300,0.000,0.030,0.000,10.000 /
&SURF ID='Wood' RGB=104,52,0 STRETCH_FACTOR=1.0 CELL_SIZE_FACTOR=0.5
      MATL_ID(1,1:3)='CELLULOSE','WATER','LIGNIN'
      MATL_MASS_FRACTION(1,1:3)=0.70,0.10,0.20 
      THICKNESS=0.03 /
```
My model looks like this:

![image](https://github.com/firemodels/fds/assets/15341622/38998e43-95e1-4c67-814d-505b9aa98bbb)

My questions are about the thickness of the plank, in relation to the mesh size.

If I include the whole building (9 floors), I will end up with a domain of about 140x90x50.

My questions are:

1. The smallest dimension in my model is 3cm. Does this mean I have to have a small mesh. If my mesh is 10cm, with the size of the domain, I will end up with 630 million cells. Do I really have to do this?

2. I am wondering what is the THICKNESS option in &SURF? How does it interact with the actual thickness of the obstruction?

3. Also, what is the option THICKEN=T? Does it relevant for this case?

4. Any suggestion on how to reduce the number of mesh would be appreciated.

Thanks

**Comment by rmcdermo** [comment]
Nice looking model...  but now you have the actual physics to deal with.

1. This depends.  It sounds like you are trying to actually model the flame spread, in which case you will probably need very fine grid resolution, maybe even sub 3 cm.  But it depends on too many factors.  We can't teach all of fire dynamics modeling in the answer to a Discussion post----you need to spend some time with the manuals.
2. See FDS user guide.
3. See FDS user guide.
4. It is not clear what you are trying to model.  If it is flame spread over the entire structure, you probably cannot reduce the cell count.  If you are just interested in the performance of the facade, you might consider only modeling a section of the building.

**Comment by mcgratta** [comment]
On this forum, the picture of your model is not the one we want to see. You should look at the rendering of the model by FDS. That will tell you more about resolution. Also, distinguish between the terms "flame spread" and "fire spread", If you are interested in the details of how the fire is going to spread underneath these wood planks, you may want to model a portion of the building at higher resolution. That is what we would call "flame spread". If, on the other hand, you want to get a sense of how a fire might be affected by these planks, and how these planks might provide more fuel for the fire, you might be able to use a coarser mesh. This is what we call "fire spread." Exactly how you do it is part of the skill of modeling. There are no clear cut rules as to how you do it.
